docs/expansionPort.md

```diff
@@ -0,0 +1,115 @@
+
+
+### 4. Single-ended to differential converter
+
+The I/O interface is connected to the FPGA I/O pins of the PicoZed, which can be configured as SPI, UART, or any other communication interface. This signal is converted into differential I/O using the AM26C31 differential line driver. The operating conditions for the IC are provided in the following table.
+
+| Parameter                             | MIN   | NOM | MAX   |
+|---------------------------------------|-------|-----|-------|
+| Supply voltage Vcc                    | 4.5 V | 5 V | 5.5 V |
+| Input voltage range                   | -0.5 V|     | Vcc + 0.5 V  |
+| High-level input voltage              | 2 V   |     | Vcc   |
+| Low-level input voltage               | 0 V   |     | 0.8 V |
+
+The maximum supply current consumed by the IC including the drive currents for the differential lines is around 20 mA, which corresponds to 100 mW for a 5 V supply. For more information refer to the [AM26C31](http://www.ti.com/lit/ds/symlink/am26c31.pdf?HQS=TI-null-null-digikeymode-df-pf-null-wwe&ts=1590045318995) datasheet.
+
+
+### 5. Differential to single-ended converter
+
+The differential signal is then converted into a single-ended signal using the AM26C32 differential line receiver. The operating conditions for the IC are provided in the following table.
+
+| Parameter                             | MIN   | NOM | MAX   |
+|---------------------------------------|-------|-----|-------|
+| Supply voltage Vcc                    | 4.5 V | 5 V | 5.5 V |
+| Common-mode input voltage             | -7 V  |     | +7 V  |
+| High-level input voltage              | 2 V   |     | Vcc   |
+| Low-level input voltage               | 0 V   |     | 0.8 V |
+
+The maximum supply current consumed by the IC including the drive currents for the differential lines is around 10 mA, which corresponds to 50 mW for a 5 V supply. For more information refer to the [AM26C32](http://www.ti.com/lit/ds/symlink/am26c32.pdf?HQS=TI-null-null-digikeymode-df-pf-null-wwe&ts=1590045351338) datasheet.
+
+
+### 6. Voltage level shifter
+
+The 5V voltage level from the differential line receiver is translated to 1.8V for the PicoZed using the SN74LVC8T245 level translation IC. This IC supports bi-directional translation and is also used to translate the low-voltage level (from PicoZed) to 5 V level used by the differential line driver. The voltage levels are translated based on the supply voltage rail A (VCCA) and supply voltage rail B (VCCB). The direction can be set using a direction control pin (DIR), a high on the DIR pin translates the signals from A to B and a low on DIR translates the signals from B to A. The operating voltage range for both A and B ports is from 1.65 V to 5.5 V.  For more information refer the IC [datasheet](http://www.ti.com/lit/ds/symlink/sn74lvc8t245.pdf?HQS=TI-null-null-digikeymode-df-pf-null-wwe&ts=1590052474879).    

```

Comment:
Is this stating that the firmware of the PicoZED can change the direction of the level shifting? If this is permanently fixed at design time, I recommend that you do not discuss it.

Reply:
The direction of the level inverter is fixed during design, I have removed this sentence.